Header header

float32 maximum_value
float32 minimum_value
